plugins/providers/hyperv/action/configure.rb

```diff
@@ -97,6 +97,11 @@ def call(env)
               env[:machine].provider_config.vm_integration_services)
           end
 
+          if env[:machine].provider_config.enable_enhanced_session_mode
```

Comment:
This looks great, just one more thing. Users should be able to disable EnhancedSessionTransportType as well as enable it. 
So something like
```
$ cat Vagrantfile
# Vagrantfile
 ...
    h.vm.provider "hyperv" do |b|
      b.enable_enhanced_session_mode = true
    end
 ...

$ vagrant up box
```
will bring up the box with the EnhancedSessionTransportType set to `HvSocket`. Then editing the Vagrantfile setting enable_enhanced_session_mode to false 
```
$ cat Vagrantfile
# Vagrantfile
 ...
    h.vm.provider "hyperv" do |b|
      b.enable_enhanced_session_mode = false
    end
  ...

$ vagrant reload box
```
should reload the box with the EnhancedSessionTransportType set to `VMBus`. However, currently it leaves the EnhancedSessionTransportType set to `HvSocket`

Reply:
Thanks for the feedback. Added this in commit. 1ed4c882dc6adc19b330da146a432265cd152a34
Note as its a bool setting to false is the same as not present, which is I think the best behavior for a new feature.

Reply:
Awesome, this looks great! Thanks 🙏